.release-please-manifest.json

```diff
@@ -1,5 +1,5 @@
 {
-  "flagd": "0.5.4",
-  "flagd-proxy": "0.2.4",
-  "core": "0.5.4"
+  "flagd": "0.5.5",
```

Comment:
Should we manually bump this version since there was a breaking change in core?

Reply:
Done. bumped to 0.6.0